tests/reftests/dune.inc

```diff
@@ -1427,6 +1427,24 @@
    %{targets}
    (run ./run.exe %{exe:../../src/client/opamMain.exe.exe} %{dep:var-option.test} %{read-lines:testing-env}))))
 
+(rule
+ (alias reftest-var-syntax)
```

Comment:
The name is not explicit, maybe `opam-var-++-syntax` or `opam-var-filter-syntax`

Reply:
opam-var-filter-syntax then